src/services/health-check/default-tagline.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Health_Check;
+
+/**
+ * Passes when the tagline is set to something other than the WordPress default tagline.
+ */
+class Default_Tagline extends Health_Check {
```

Comment:
```suggestion
class Default_Tagline_Check extends Health_Check {
```
ik mis het werkwoord. `Default_Tagline` alleen betekent niks.

Reply:
@diedexx same here. wat is beter?

Reply:
Agreed!  It's a specific type of check, so the name should reflect that